dist/data/valacdos-v440.txt

```diff
@@ -1,4 +1,7 @@
 010004 016715 MOD11    0    0    0    0    0    0    8    7    6    5    4    3    2    1

```

Comment:
The `dist` files are only added at release time. Please remove `dist/data/valacdos-v440.txt`.